Makefile

```diff
@@ -143,10 +148,82 @@ generate: $(TOOLS_DIR)/stringer
 
 .PHONY: license-check
 license-check:
-	@licRes=$$(for f in $$(find . -type f \( -iname '*.go' -o -iname '*.sh' \) ! -path './vendor/*') ; do \
+	@licRes=$$(for f in $$(find . -type f \( -iname '*.go' -o -iname '*.sh' \) ! -path './vendor/*' ! -path './internal/opentelemetry-proto/*') ; do \
 	           awk '/Copyright The OpenTelemetry Authors|generated|GENERATED/ && NR<=3 { found=1; next } END { if (!found) print FILENAME }' $$f; \
 	   done); \
 	   if [ -n "$${licRes}" ]; then \
 	           echo "license header checking failed:"; echo "$${licRes}"; \
 	           exit 1; \
 	   fi
+
+# Find all .proto files.
+OTEL_PROTO_SUBMODULE := internal/opentelemetry-proto
+PROTO_GEN_DIR        := internal/opentelemetry-proto-gen
+PROTOBUF_TEMP_DIR    := gen/pb-go
+PROTOBUF_SOURCE_DIR  := gen/proto
+ORIG_PROTO_FILES     := $(wildcard $(OTEL_PROTO_SUBMODULE)/opentelemetry/proto/*/v1/*.proto \
+                           $(OTEL_PROTO_SUBMODULE)/opentelemetry/proto/collector/*/v1/*.proto)
```

Comment:
Might want to make the `v1` version here a variable as well.

Reply:
Done.